### PR TITLE
Seed new policies

### PR DESCRIPTION
**Pull request by tekin**
**NOTE: DO NOT MERGE BEFORE https://trello.com/c/ql7lHif8/181-add-feature-flagging-to-policy-publisher**

Migrations to seed the new policies into policy publisher. This should not be merged until after the [feature-switch code](https://github.com/alphagov/policy-publisher/pull/42) to ensure that the policies are not pushed out onto GOV.UK before we intend them to be.

Trello: https://trello.com/c/Gl4M63fv/182-seed-policy-publisher-with-new-policies

### CODE REVIEW

File: db/migrate/20150416085800_associate_parent_policies.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class AssociateParentPolicies < ActiveRecord::Migration
+  POLICY_PARENT_MAPPINGS = {
+    'automatic-enrolment-in-workplace-pensions' => ['employment', 'older-people'],
+    'city-deal' => ['localism'],
+    'regional-growth-fund' => ['employment'],
+    'hs2-high-speed-rail' => ['rail-network']
```
Comment (elliotcm) [inline db/migrate/20150416085800_associate_parent_policies.rb]: Are we certain enough that these slugs will be created?  Should we use content IDs instead?

File: db/migrate/20150416085800_associate_parent_policies.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  }
+
+  def change
+    POLICY_PARENT_MAPPINGS.each do |policy_slug, parent_policy_slugs|
+      policy = Policy.find_by!(slug: policy_slug)
+      parents = Policy.where(slug: parent_policy_slugs)
+      policy.parent_policies = parents
```
Comment (elliotcm) [inline db/migrate/20150416085800_associate_parent_policies.rb]: You don't save the policy here.

Comment (tekin) [inline db/migrate/20150416085800_associate_parent_policies.rb]: It's not necessary for the purposes of saving the association as the join records are created with the assignment.

File: db/migrate/20150416085800_associate_parent_policies.rb (hunk 1 above)
Comment (tekin) [inline db/migrate/20150416085800_associate_parent_policies.rb]: If they don't, there will be an exception raised.

File: db/migrate/20150416085800_associate_parent_policies.rb (hunk 2 above)
Comment (elliotcm) [inline db/migrate/20150416085800_associate_parent_policies.rb]: Rails :/

File: db/migrate/20150416085800_associate_parent_policies.rb (hunk 1 above)
Comment (elliotcm) [inline db/migrate/20150416085800_associate_parent_policies.rb]: OK.

**Pull request merged**